model/postgres/record.py

```diff
@@ -105,8 +122,11 @@ def get_parts(self) -> list[Part]:
 
         for part in self.has_part:
             index, file_url, source, file_type, flags = part.split('|')
+            
+            if not index:
+                index = 0
```

Comment:
Will this be an issue if there are multiple parts without an index?

Reply:
Possibly - yes if you use the get_parts function - lemme just make it optional